src/components/TileCatalog/TitleCatalog.story.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { number, boolean } from '@storybook/addon-knobs';
+
+import TileCatalog from './TileCatalog';
+import SampleTile from './SampleTile';
+
+const sortOptions = [
+  { text: 'Choose from options', id: 'Choose from options' },
+  { text: 'A-Z', id: 'A-Z' },
+  { text: 'Most Popular', id: 'Most Popular' },
+];
+const selectedSortOption = 'Choose from options';
+
+const getTiles = num => {
+  const tiles = [];
+  Array(num)
+    .fill(0)
+    .map((i, idx) => {
+      tiles[idx] = <SampleTile title={`${idx + 1}`} description="This is a sample product tile" />;
+      return tiles[idx];
+    });
+  return tiles;
+};
+
+storiesOf('Watson IoT Experimental|TileCatalog', module)
+  .add('Base', () => {
+    const numOfTiles = number('number of tiles', 10);
+    return (
+      <div style={{ width: '60rem' }}>
+        <TileCatalog
+          title="Product name"
+          tiles={getTiles(
+            numOfTiles,
+            <SampleTile title="Sample product tile" description="This is a sample product tile" />
+          )}
+          numColumns={number('numColumns', 4)}
+          numRows={number('numRows', 2)}
+          hasSearch={boolean('hasSearch', true)}
+          hasSort={boolean('hasSort', true)}
+        />
+      </div>
+    );
+  })
+  .add('With Search', () => (
+    <div style={{ width: '60rem' }}>
+      <TileCatalog
+        title="Product name"
+        tiles={getTiles(20)}
+        numColumns={number('numColumns', 4)}
+        numRows={number('numRows', 2)}
+        hasSearch={boolean('hasSearch', true)}
+        onSearch={action('search', () => {})}
+      />
+    </div>
+  ))
+  .add('With Sort', () => (
+    <div style={{ width: '60rem' }}>
+      <TileCatalog
+        title="Product name"
+        tiles={getTiles(
+          8,
+          <SampleTile title="Sample product tile" description="This is a sample product tile" />
+        )}
+        numColumns={number('numColumns', 4)}
+        numRows={number('numRows', 2)}
+        hasSort={boolean('hasSort', true)}
+        sortOptions={sortOptions}
+        onSort={action('sort', () => {})}
+        selectedSortOption={selectedSortOption}
+      />
+    </div>
+  ))
+  .add('StatefulTileCatalog', () => {
+    const StatefulTileCatalog = () => {
```

Comment:
Should we export this for others to use?

Reply:
I don't think so; the new TileCatalog doesn't include a data model for the individual tiles (just the node to be rendered) so any sort and search logic has to be implemented by the user directly.  TileCatalog does handle pagination in component state, so it's kind of a "half-stateful" component.

Reply:
This is just an example not for others to use.